ugs-core/src/com/willwinder/universalgcodesender/MacroHelper.java

```diff
@@ -22,30 +22,18 @@ This file is part of Universal Gcode Sender (UGS).
 import com.willwinder.universalgcodesender.model.BackendAPI;
 import com.willwinder.universalgcodesender.model.Position;
 import com.willwinder.universalgcodesender.services.KeyboardService;
-import com.willwinder.universalgcodesender.utils.GUIHelpers;
 import net.miginfocom.swing.MigLayout;
 import org.apache.commons.lang3.RegExUtils;
 import org.apache.commons.lang3.StringUtils;
 
-import javax.swing.JLabel;
-import javax.swing.JOptionPane;
-import javax.swing.JPanel;
-import javax.swing.JTextField;
-import javax.swing.KeyStroke;
-import java.awt.AWTException;
-import java.awt.Robot;
-import java.awt.event.InputEvent;
+import javax.swing.*;
```

Comment:
This is a minor thing. Can you change the settings in your IDE to not import *.
This gets annoying when for instance I want to import `java.util.List` but instead get a `javax.awt.List`.